README.md

```diff
@@ -0,0 +1,273 @@
+instances created by the two versions returns `false`, and if the export is an
+object, properties added to one (like `pkgInstance.foo = 3`) are not present on
+the other. This differs from how `import` and `require` statements work in
+all-CommonJS or all-ES module environments, respectively, and therefore is
+surprising to users. It also differs from the behavior users are familiar with
+when using transpilation via tools like [Babel][] or [`esm`][].
+
+### Writing dual packages while avoiding or minimizing hazards
+
+First, the hazard described in the previous section occurs when a package
+contains both CommonJS and ES module sources and both sources are provided for
+use in Node.js, either via separate main entry points or exported paths. A
+package might instead be written where any version of Node.js receives only
+CommonJS sources, and any separate ES module sources the package might contain
+are intended only for other environments such as browsers. Such a package
+would be usable by any version of Node.js, since `import` can refer to CommonJS
+files; but it would not provide any of the advantages of using ES module syntax.
+
+A package might also switch from CommonJS to ES module syntax in a [breaking
+change](https://semver.org/) version bump. This has the disadvantage that the
+newest version of the package would only be usable in ES module-supporting
+versions of Node.js.
+
+Every pattern has tradeoffs, but there are two broad approaches that satisfy the
+following conditions:
+
+1. The package is usable via both `require` and `import`.
+2. The package is usable in both current Node.js and older versions of Node.js
+   that lack support for ES modules.
+3. The package main entry point, e.g. `'pkg'` can be used by both `require` to
+   resolve to a CommonJS file and by `import` to resolve to an ES module file.
+   (And likewise for exported paths, e.g. `'pkg/feature'`.)
+4. The package provides named exports, e.g. `import { name } from 'pkg'` rather
+   than `import pkg from 'pkg'; pkg.name`.
+5. The package is potentially usable in other ES module environments such as
+   browsers.
+6. The hazards described in the previous section are avoided or minimized.
+
+#### Approach #1: Use an ES module wrapper
+
+Write the package in CommonJS or transpile ES module sources into CommonJS, and
+create an ES module wrapper file that defines the named exports. Using
+[Conditional exports][], the ES module wrapper is used for `import` and the
+CommonJS entry point for `require`.
+
+```json
+// ./node_modules/pkg/package.json
+{
+  "type": "module",
+  "exports": {
+    "import": "./wrapper.mjs",
+    "require": "./index.cjs"
+  }
+}
+```
+
+The preceding example uses explicit extensions `.mjs` and `.cjs`.
+If your files use the `.js` extension, `"type": "module"` will cause such files
+to be treated as ES modules, just as `"type": "commonjs"` would cause them
+to be treated as CommonJS.
+See [Enabling](esm.md#enabling).
```

Comment:
This link needs to be corrected to https://nodejs.org/api/esm.html#enabling

Reply:
Thanks for catching it, fixed.